src/main/java/org/qe/hawkular/page/HawkularCreateCasePage.java

```diff
@@ -0,0 +1,87 @@
+package org.qe.hawkular.page;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.qe.hawkular.element.HawkularCreateCaseConstants;
+import org.qe.hawkular.util.HawkularUtils;
+import org.testng.Assert;
+
+public class HawkularCreateCasePage {
+
+	public final WebDriver driver;
+
+	public HawkularCreateCasePage(WebDriver driver) {
+		this.driver = driver;
+	}
+
+	By clickRedHatAccess = HawkularCreateCaseConstants.clickRedHatAccess;
+	By clickRedHatAccessSupport = HawkularCreateCaseConstants.clickRedHatAccessSupport;
+	By clickRedHatAccessSupportOpenCase = HawkularCreateCaseConstants.clickRedHatAccessSupportOpenCase;
+	By enterRedHatAccessAccount = HawkularCreateCaseConstants.enterRedHatAccessAccount;
+	By clickRedHatProduct = HawkularCreateCaseConstants.clickRedHatProduct;
+	By selectRedHatProduct = HawkularCreateCaseConstants.selectRedHatProduct;
+	By selectRedHatProductVersion = HawkularCreateCaseConstants.selectRedHatProductVersion;
+	By typeSummary = HawkularCreateCaseConstants.typeSummary;
+	By writeDescription = HawkularCreateCaseConstants.writeDescription;
+	By clickNext = HawkularCreateCaseConstants.clickNext;
+	By clickSupportLevel = HawkularCreateCaseConstants.clickSupportLevel;
+	By selectSupportLevel = HawkularCreateCaseConstants.selectSupportLevel;
+	By clickSeverity = HawkularCreateCaseConstants.clickSeverity;
+	By selectSeverity = HawkularCreateCaseConstants.selectSeverity;
+	By clickSubmit = HawkularCreateCaseConstants.clickSubmit;
+
+
```

Comment:
@skondkar I feel, creating local reference is not require, you can call directly from `HawkularCreateCaseConstants`